src/lib/resource/sections/applicant-select-reasons-for-the-delay-in-making-your-application.js

```diff
@@ -51,6 +53,10 @@ module.exports = {
                                 'Briefly explain reasons for the delay in making your application'
```

Comment:
Error message to match questions - Tell us the reason why this application was not sent until now